src/main/res/drawable/ic_size_largest.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="23dp" android:viewportHeight="512"
```

Comment:
23dp seems like a very random value. Is it generated by Android Studio only? Usually, we go in steps like 18, 24, 36, 48dp for icon sizes.